aleph-client/src/lib.rs

```diff
@@ -160,3 +161,39 @@ pub fn keypair_from_string(seed: &str) -> KeyPair {
 pub fn account_from_keypair(keypair: &KeyPair) -> AccountId {
     AccountId::from(keypair.public())
 }
+
+fn storage_key(module: &str, version: &str) -> [u8; 32] {
+    let pallet_name = sp_io::hashing::twox_128(module.as_bytes());
+    let postfix = sp_io::hashing::twox_128(version.as_bytes());
+    let mut final_key = [0u8; 32];
+    final_key[..16].copy_from_slice(&pallet_name);
+    final_key[16..].copy_from_slice(&postfix);
+    final_key
+}
+
+/// Computes hash of given pallet's call. You can use that to pass result to `state.getKeys` RPC call.
+/// * `pallet` name of the pallet
+/// * `call` name of the pallet's call
+///
+/// # Example
+/// ```
+/// let staking_nominate_storage_key = get_storage_key("Staking", "Nominators");
+/// assert_eq!(staking_nominate_storage_key, String::from("5f3e4907f716ac89b6347d15ececedca9c6a637f62ae2af1c7e31eed7e96be04"));
+/// ```
+pub fn get_storage_key(pallet: &str, call: &str) -> String {
+    let bytes = storage_key(pallet, call);
+    let storage_key = StorageKey(bytes.into());
+    hex::encode(storage_key.0)
+}
+
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn given_staking_nominate_call_when_get_storage_key_then_hash_is_correct() {
```

Comment:
I'm not sure if this test adds much value - and examples in the docstring can be run as tests anyway

Reply:
Right, will remove UT